cmake-modules/AddTestCMocka.cmake

```diff
@@ -1,39 +1,15 @@
+#
+# Copyright (c) 2007      Daniel Gollub <dgollub@suse.de>
```

Comment:
Has CMocka been entered into the component governance site.  Is this copyright usage approved?

Reply:
will add the Notice and cgmanifest on https://github.com/Azure/azure-sdk-for-c/pull/1213

This PR is just to update the licenses in cmake modules.  Once this is merged, I will update the other PR to have all the Notices